### ISSUE TITLE
objcopy: command not found

### ISSUE THREAD

**Issue by sinkarharshad** [RIOT-OS/RIOT#5960]
Hi,

I'm trying to setup RIOT shell on MAC OSX using this guide: http://riot-os.org/api/index.html#the-quickest-start

While trying 'make all' I'm getting an error:

__TEXT  __DATA  __OBJC  others  dec hex
90112   1220608 0   24576   1335296 146000  
bash: objcopy: command not found
make: **\* [all] Error 127

And there's an error after using the command: 'git checkout <LATEST_RELEASE>'
Error: -bash: syntax error near unexpected token `newline'

I installed binutils but the package name is gobjcopy rather than objcopy. I'm unable to use it. 
I also added alias in ~/.profile as:

alias objcopy="gobjcopy"
alias objdump="gobjdump"

This also didn't work and I got the same error that objdump command is not found.

Please let me know how to proceed from here

**Comment by Yonezawa-T2** [comment]
If you are using MacPorts, setting `PATH` will work.
`PATH=$PATH:/opt/local/x86_64-apple-darwin15.6.0/bin/ make BOARD=native`

**Comment by sinkarharshad** [comment]
I tried, and still I get the same error. ): 
![screen shot 2016-10-18 at 12 21 05](https://cloud.githubusercontent.com/assets/18270841/19473809/68616454-952d-11e6-8652-7456f6878d4d.png)

**Comment by Yonezawa-T2** [comment]
Type `mdfind -name objcopy` to locate where the `objcopy` is installed.

If you don't have a file named `objcopy` but `gobjcopy`, make a symlink to it and add it to your `PATH`:
```
mkdir -p ~/bin
ln -s /path/to/gobjcopy ~/bin/objcopy
ln -s /path/to/gobjdump ~/bin/objdump
...
 PATH=$PATH:~/bin/ make BOARD=native
```

**Comment by sinkarharshad** [comment]
I have installed objcopy. what should i do next? (Below is the path)
![screen shot 2016-10-18 at 12 33 12](https://cloud.githubusercontent.com/assets/18270841/19474253/238e17bc-952f-11e6-8b38-f4167b16978a.png)

**Comment by Yonezawa-T2** [comment]
```
mkdir -p ~/bin
ln -s /usr/local/Cellar/binutils/2.27/bin/gobjcopy ~/bin/objcopy
PATH=$PATH:~/bin make BOARD=native
```

**Comment by Yonezawa-T2** [comment]
To other maintainers:
BTW, `boards/native/Makefile.include` will use `gobjcopy` if it is available, but `cpu/Makefile.include.llvm` just try to use `objcopy`. I guess that is the cause of this issue (not confimed yet). If so, we should update `cpu/Makefile.include.llvm` to use `gobjcopy` if `objcopy` is not available.

**Comment by sinkarharshad** [comment]
After following  above instructions, again when i tried running "make all" i still got the same error.  :! (or after following above instruction, I don't need to run "make all" ? )

**Comment by kYc0o** [comment]
I confirm the issue, I didn't provide a patch for that since I don't use a lot the native port but I think we should work on it, even if there are no "a lot" of Mac OS X users I think is important.

**Comment by miri64** [comment]
@kYc0o would it be possible to provide this patch for the release?

**Comment by Yonezawa-T2** [comment]
@sinkarharshad 
What is displayed with the following commands?

```bash
ls -l ~/bin
export PATH=$PATH:~/bin
type objcopy
```

**Comment by Yonezawa-T2** [comment]
@miri64 @kYc0o I made a PR #5965.

**Comment by sinkarharshad** [comment]
@Yonezawa-T2 "make all" no error, for "make term" I am getting this following error. 
![screen shot 2016-10-19 at 05 40 56](https://cloud.githubusercontent.com/assets/18270841/19504952/a73848f0-95be-11e6-8b5b-3c96e703f450.png)

**Comment by Yonezawa-T2** [comment]
@sinkarharshad Install http://tuntaposx.sourceforge.net/.

**Comment by sinkarharshad** [comment]
After installing tunetap, and I tried again .. and got following error. 
![screen shot 2016-10-19 at 06 00 18](https://cloud.githubusercontent.com/assets/18270841/19505270/75099b74-95c1-11e6-8ace-81cb23655d01.png)

**Comment by Yonezawa-T2** [comment]
Try `sudo make term`.

Tips: if you get `resource busy` error on tap interface, find a process named *.elf, then kill it.

**Comment by sinkarharshad** [comment]
It worked, and finally got a message saying .. "Welcome to RIOT" .. thank you all for religiously putting your efforts helping me. Thank you one more time. Cheers! Also, As I am beginner to riot, can you please guide me? How should i start writing simple programs on RIOT? Thank you!

**Comment by miri64** [comment]
https://github.com/RIOT-OS/RIOT/pull/5965 is not merged yet

**Comment by diskmila** [comment]
Any updates on this matter? I'm having the same problem

**Comment by kYc0o** [comment]
So far it was fixed on #5965, what's your exact problem?

**Comment by diskmila** [comment]
when running make all and make term from the tutorial I get the objcopy not found error
![Captura de Tela 2022-05-04 às 14 38 17](https://user-images.githubusercontent.com/81367935/166746737-6c660478-f675-46c1-aeb7-967c19f43db0.png)
